logging_utils.py

```diff
@@ -27,14 +28,19 @@ def get_error_logger(action_type, object_type, log_dir):
     logger.addHandler(error_handler)
     return logger
 
+
 def get_error_log_file(action_type, object_type, parent_dir):
     return f"{_get_log_dir(parent_dir)}/failed_{action_type}_{object_type}.log"
 
+
 def _get_log_dir(parent_dir):
     return parent_dir + "/app_logs"
 
-def log_reponse_error(error_logger, response, error_msg=None):
-    if 'error_code' in response or 'error' in response:
+
+def log_reponse_error(error_logger, response, error_msg=None, error_ignore_list=[]):
```

Comment:
If we are intending to ignore the ALREADY_EXIST error in the below logic, should we just put it into the default value of the error_ignore_list? (['RESOURCE_ALREADY_EXISTS'])

Reply:
done